bin/lsdcRemote

```diff
@@ -0,0 +1 @@
+script -a -c lsdcRemote.cmd -f lsdcRemoteLog.txt
```

Comment:
I'm not super familiar with the script command, but I *think* this will try to run the "lsdcRemote.cmd" file which is now missing (having been renamed to "lsdcRemote_amx.cmd" and "lsdcRemote_fmx.cmd").

Reply:
The procedure for creating the symlinks will help explain this. Probably should be documented!!

On both AMX and FMX, the lsdcServer and lsdcRemote commands are symlinked from /usr/local/bin without any changes to filename.

For the scripts lsdcServer_amx/fmx.cmd and lsdcRemote_amx/fmx.cmd, these files are symlinked and renamed to lsdcServer.cmd and lsdcRemote.cmd, as appropriate for the beamline.

This allows the same general script to be used for both/all beamlines. Eventually this should be added to documentation, and implemented in ansible jobs for installing LSDC